kahuna/public/js/components/gr-image-usage/gr-image-usage.js

```diff
@@ -58,12 +58,19 @@ module.directive('grImageUsage', [function() {
     };
 }]);
 
-module.controller('grImageUsageListCtrl', [function () {
+module.controller('grImageUsageListCtrl', [
+    'imageUsagesService',
+    function (imageUsagesService) {
     const ctrl = this;
 
     ctrl.formatTimestamp = (timestamp) => {
         return moment(timestamp).fromNow();
     };
+
+    ctrl.isRecent = (timestamp) => {
+        var nowtime = new Date();
```

Comment:
This file seems to be using ES6 const/let. I think this should probably be `const` to match style